package.json

```diff
@@ -34,15 +34,15 @@
   "module": "./lib/index.js",
   "types": "./dist/index.d.ts",
   "dependencies": {
-    "@formatjs/intl-relativetimeformat": "^2.6.2",
+    "@formatjs/intl-relativetimeformat": "^2.6.3",

```

Comment:
I included those deps updates as I saw they were freshly published. Let me know if it's ok, otherwise I can revert.